README.md

```diff
@@ -12,34 +12,20 @@ The easiest way to get a production grade Kubernetes cluster up and running.
 
 We like to think of it as `kubectl` for clusters.
 
-`kops` lets you deploy production-grade Kubernetes clusters from the command line, with
-options that support HA Masters. Kubernetes Operations supports deploying
-Kubernetes on Amazon Web Services (AWS) and support for more platforms is planned.
-
-## Launching a Kubernetes cluster hosted on AWS
-
-Check out our [tutorial](/docs/aws.md) for launching a Kubernetes cluster hosted
-on AWS.
-
-<p align="center">
-  <a href="https://asciinema.org/a/97298">
-  <img src="https://asciinema.org/a/97298.png" width="885"></image>
-  </a>
-</p>
-
+`kops` lets you deploy production-grade, with highly available masters,
+Kubernetes clusters from the command line.  Deployment is currently supported
+on Amazon Web Services (AWS), with more platforms planned.
 
 ## Features
 
-* Automated Kubernetes cluster [CRUD](/docs/commands.md) for ([AWS](/docs/aws.md))
-* Highly Available (HA) Kubernetes Masters Setup
-* Uses a state-sync model for **dry-run** and automatic **idempotency**
-* Custom support for `kubectl` [add-ons](/docs/addons.md)
-* Kops can generate [Terraform configuration](/docs/terraform.md)
-* Based on a simple meta-model defined in a directory tree
-* Command line [autocomplete](/docs/cli/kops_completion.md)
-* Community support
-* [Upgrade from kube-up](/docs/upgrade_from_kubeup.md)
-
+* Automate the provisioning of Kubernetes clusters in ([AWS](/docs/aws.md))
```

Comment:
Hmm, I think one of the benefits is you can easily control multiple clusters, so singular would seem a bit weird.  We're going to have qa/staging/prod clusters and we can't be too unique!